Libraries/Components/View/ViewPropTypes.js

```diff
@@ -352,6 +352,19 @@ type AndroidViewProps = $ReadOnly<{|
    */
   focusable?: boolean,
 
+  /**
+   * Indicates whether this `View` should be focusable with a non-touch input device, eg. receive focus with a hardware keyboard.
+   * See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex
+   * for more details.
+   *
+   * Supports the following values:
+   * - -1 (View is focusable)
+   * - 0 (View is not focusable)
```

Comment:
This comment is incorrect, `0` is focusable and `-1` is not focusable

Reply:
Fixed